app/routes/edit-account.js

```diff
@@ -0,0 +1,7 @@
+import AuthenticatedRoute from 'ember-authenticate-me/routes/authenticated';
+
+export default AuthenticatedRoute.extend({
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.